apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -90,8 +91,11 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
 
         const properties = Object.keys(document?.propertiesSchema?.properties ?? {}).reduce(
```

Comment:
Alternative for the same result is about to run over entries instead of manipulating the object's keys 
```suggestion
const properties = Object.entries(document?.propertiesSchema?.properties ?? {}).reduce(
  (acc, [curr, schema]) => {
    const formDataValue = formData?.[curr];

    // Check if form data value is nullish; if so, skip this property
    if (isNullish(formDataValue)) return acc;

    // Serialize the form data value using the schema
    acc[curr] = serializeValue(formDataValue, schema);

    return acc;
  },
  {}
);
```